.github/workflows/reusable-dev-deployment.yml

```diff
@@ -60,7 +60,7 @@ jobs:
             |<span aria-hidden="true">🔨</span> Latest commit | [${{ inputs.sha }}](https://github.com/${{ github.repository }}/commit/${{ inputs.sha }}) |
```

Comment:
If you want to test changes to the deployment, you need to push to a branch in ecamp directly, and then create a PR on this new branch.
Or you make the links the default and then you don't need to change the workflow files

Reply:
Done with #4776, it works